src/components/SearchBar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function SearchBar({ query, setQuery, callback }) {
+  return (
+    <div>
+      <input
+        value={query}
+        onChange={(e) => setQuery(e.target.value)}
+        placeholder="Search Pokemon Name"
+      />
+      <button onClick={callback}>Submit</button>
```

Comment:
You have this `onClick` that is a suppose to act like a submit button but when I query a pokemon it changing with each character. You can change the `onClick` to take in the value vs. the `onChange` if that is how you want it work.